libs/connectors_sdk/pyproject.toml

```diff
@@ -0,0 +1,51 @@
+[build-system]
+requires = ["setuptools"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "elasticsearch-connectors-sdk"
+description = "Elasticsearch Connectors SDK"
+dynamic = ["version"]
+readme = "README.md"
+authors = [
+    { name = "Search Extract and Transform Team", email = "enterprise-search-ingestion-team@elastic.co" }
+]
+classifiers = [
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python :: 3 :: Only",
+]
+requires-python = ">=3.10,<3.12"
+dependencies = [
+    "aiofiles==23.2.1",
+    "aiohttp==3.12.14",
+    "base64io==1.0.3",
+    "fastjsonschema==2.16.2",
+    "ecs-logging==2.0.0",
+    "elasticsearch"
```

Comment:
I didn't see any `elasticsearch` imports in connectors_sdk, we can remove it. Same for `fastjsonschema`, maybe the dependency reduction was WIP.

Reply:
Lol that was fast. I added it cause of random warning from `pytest` cause it failed to run.

There was a warning exception in `pytest.ini` in the root of the repo and it was mentioning a warning in `elasticsearch`:

```
ignore::elasticsearch.exceptions.GeneralAvailabilityWarning
```

SInce SDK did not have this module. pytest was dying. I added the line to continue and just dropped it in a PR

`fastjsonschema` comes here: https://github.com/elastic/connectors/blob/0954997ec24194c67b459e27abd04ed0df0300a5/libs/connectors_sdk/connectors_sdk/filtering/validation.py#L9